### PR TITLE
Update loading spec to impl b2 snapshot

### PR DESCRIPTION
**Pull request by nyaxt**
***
<a href="https://pr-preview.s3.amazonaws.com/nyaxt/webpackage/pull/328.html" title="Last updated on Dec 18, 2018, 12:01 AM UTC (b2687a5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/webpackage/328/9b58b4c...nyaxt:b2687a5.html" title="Last updated on Dec 18, 2018, 12:01 AM UTC (b2687a5)">Diff</a>

### CODE REVIEW

**Comment by twifkak** [comment]
ref #310

File: loading.bs [hunk 1]
```diff
@@ -440,7 +440,7 @@ following steps:
 1. Let |stream| be a [=new read buffer=] for |bodyStream|.
 1. Let |magic| be the result of [=read buffer/reading=] 8 bytes from |stream|.
 1. If |magic| is a failure, return it.
-1. If |magic| is not `` `sxg1-b1\0` ``, return a failure.
+1. If |magic| is not `` `sxg1-b2\0` ``, return a failure.
 1. Let |encodedSigLength| be the result of [=read buffer/reading=] 3 bytes from
```
Comment (jyasskin) [inline loading.bs]: b2 includes a fallback URL here. https://tools.ietf.org/html/draft-yasskin-httpbis-origin-signed-exchanges-impl-02#section-5.3

**Comment by nyaxt** [comment]
I'll address your comments shortly, but feel free to take over this change (or any of my PRs when you think it would be more productive).
I'm happy with both having you pushing new changes to this PR branch or creating a new PR from scratch.

File: loading.bs [hunk 2]
```diff
@@ -583,17 +583,17 @@ returns the result of the following steps:
 
 <h3 algorithm id="signed-message">The signed message</h3>
 
-The <dfn>b1 signed message</dfn> for an [=exchange signature=] |signature| and a
+The <dfn>b2 signed message</dfn> for an [=exchange signature=] |signature| and a
```
[the remark this answers is not in the document]
Comment (nyaxt) [reply loading.bs]: Ack. (Can address tomorrow, or feel free to take over)

[the remark this answers is not in the document]
Comment (nyaxt) [reply loading.bs]: Updated.

File: loading.bs (hunk 1 above)
Comment (jyasskin) [reply loading.bs]: The UTF-8 conversion only wound up in one of the two places, so I've refactored them together.

**Pull request merged**